src/components/atomic/audio-player/audio-player.scss

```diff
@@ -7,6 +7,7 @@
 	position: relative;
 	width: 42px;
 	border: 0;
+	outline: none;
```

Comment:
omg.  no.  
Huge accessibility red flag.  How do I know where I am on the screen?

Reply:
That actually wasn't intended to go in, needed it for testing. I've reverted!